src/js/controllers/clipboard_controller.js

```diff
@@ -0,0 +1,33 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  svgCopy =
```

Comment:
Not sure I like the SVG being inlined in the code (the build process could always bundle it) -- also, what is the licensing on these icons?

Reply:
Will look into the SVG inline. 

When it comes to licensing, I don't really know. I used [this article](https://simplernerd.com/hugo-add-copy-to-clipboard-button/) as a reference and the author uses those (both in the article itself and on his page).

Reply:
Okay, let's avoid SVG images of unknown provenance. Could you please find some permissively licensed icons and either add them as a dep, or add them as files with license comments and inline them in the build?

Reply:
Changed SVGs to ones from Google Fonts (free to use in open source, checked) and moved them to files imported to Controller.

Reply:
Code looks good -- let's just add comments to the SVG files with the license.